marda_registry/data/extractors/galvani.yml

```diff
@@ -0,0 +1,35 @@
+---
+id: >-
+    galvani
+name: >-
+    Galvani
+description: >-
+    Read proprietary file formats from electrochemical test stations, including
+    BioLogic MPR and MPT files, and the res format from Arbin.
+supported_filetypes:
+    - id: biologic-mpr
+      description: >-
+          Note: support for MPR files created by ECLab >= 11.50 requires 
+          `galvani>=0.4`.
+license:
+    spdx: GPL-3.0-only
+subject:
+    - electrochemistry
+    - voltammetry
+    - impedance spectropscopy
+citations:
+    - uri: https://github.com/echemdata/galvani
+      creators:
+          - C. Kerr
+      title: galvani github repository
+      type: software
+source_repository: https://github.com/echemdata/galvani
+usage:
+    - method: python
+      setup: galvani
+      command: galvani.BioLogic.MPRFile({{ input_path }})
```

Comment:
Wanna add the `mpt` here now or later?

Reply:
I see your comments in https://github.com/marda-alliance/metadata_extractors_registry/pull/76#discussion_r1518825453, so ignore :innocent: